src/content/tutorials/observability/data-ingestion/logs/_index.md

```diff
@@ -19,9 +19,47 @@ By default, Giant Swarm clusters starting from CAPA v29.2.0, and CAPZ v29.1.0 ar
 
 The observability platform allows to ingest logs from your workloads in a self-service way using [PodLogs][1] to select which pods to ingest logs.
 
+From release `v29.5.0` in CAPA and CAPZ, and `v29.2.0` in CAPV and CAPVCD, additional targets to collect logs from can be configured either with a pod label or a [PodLogs][1] resource.
+
+### Using pod labels
+
+The Observability Platform automatically ingest all logs for pods that are labelled with the `observability.giantswarm.io/tenant` label via PodLogs resources.
+
+Example:
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  labels:
+    app: nginx
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        # This is where the tenant label must be configured
+        observability.giantswarm.io/tenant: checkout
```

Comment:
Maybe it's better to call this tenant "my-team" ? I'm not sure we're in the context of an online shop for the examples here that this would be clear, are we? Sorry for throwing this curveball in slack :D

Reply:
For once I tried to use a real example :D I'll change that